libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -732,7 +740,47 @@ async def adelete(
         return not any(
             item.get("delete", {}).get("error") for item in response["items"]
         )
-
+    
+    def create_search_pipelines(
+        self,
+        pipeline_name: str,
+        keyword_weight: float = 0.7,
+        vector_weight: float = 0.3,
+    )-> requests.Response:
+        path=f"_search/pipeline/{pipeline_name}"
+        auth = self.http_auth
+        url = f"{self.opensearch_url}/{path}"
+
+        payload = {
+        "description": "Post processor for hybrid search",
+        "phase_results_processors": [
+            {
+            "normalization-processor": {
+                "normalization": {
+                "technique": "min_max"
+                },
+                "combination": {
+                "technique": "arithmetic_mean",
+                "parameters": {
+                    "weights": [
+                    keyword_weight,
+                    vector_weight
+                    ]
+                }
+                }
+            }
+            }
+        ]
+        }
+        try:
+            response = requests.put(url, auth=auth, json=payload, verify=False)
```

Comment:
why is this not using the existing opensearch client?

Reply:
I couldn't find any OpenSearch client method to configure/create a search pipeline. I found this https://opensearch-project.github.io/opensearch-py/api-ref/clients/ingest_client.html#opensearchpy.client.ingest.IngestClient.put_pipeline  
In their documentation, they stated that put_pipeline() is used to create or update pipelines. 
I tried but I'm getting some exceptions. 

I will post the exception I'm getting. let me know if you know how to do it!

Reply:
Hey @eyurtsev I found a way to do it! I will update the code and let you know.